common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java

```diff
@@ -70,6 +70,11 @@
 
   private static final String RECOVERY_FILE_NAME = "registeredExecutor.ldb";
 
+  // Whether failure during service initialization should stop the NM.
+  @VisibleForTesting
+  static final String STOP_ON_FAILURE_KEY = "spark.yarn.shuffle.stopOnFailure";
```

Comment:
I looked around but couldn't find any documentation specific to the YARN shuffle service, but let me look again.